pom.xml

```diff
@@ -129,6 +130,11 @@
             <artifactId>simple-omero-client</artifactId>
             <version>${soc.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.openmicroscopy</groupId>
+            <artifactId>omero-gateway</artifactId>
```

Comment:
Not necessary.

Reply:
done in https://github.com/GReD-Clermont/omero_macro-extensions/pull/10/commits/c76bd5864f85c8febd75677f0d289ae3ec3c4c20